daemon/src/oracle.rs

```diff
@@ -173,12 +188,11 @@ fn next_24_hours(datetime: OffsetDateTime) -> Vec<OffsetDateTime> {
 /// Construct the URL of `olivia`'s `BitMEX/BXBT` event to be attested
 /// for at the time indicated by the argument `datetime`.
 fn event_url(datetime: OffsetDateTime) -> String {
-    let formatter = format_description::parse(OLIVIA_EVENT_TIME_FORMAT).expect("valid formatter");
-    datetime
-        .format(&formatter)
+    let datetime = datetime
+        .format(&OLIVIA_EVENT_TIME_FORMAT)
         .expect("valid formatter for datetime");
 
-    format!("{}/BitMEX/BXBT/{}/price", OLIVIA_URL, datetime)
```

Comment:
@luckysori turns out one needs to assign variables in order to change them haha :see_no_evil: